prowler/providers/aws/services/acm/acm_certificates_rsa_key_length/acm_certificates_rsa_key_length.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "acm_certificates_rsa_key_length",
+  "CheckTitle": "Check if ACM Certificates use a secure RSA key size",
+  "CheckType": [
+    "Data Protection"
+  ],
+  "ServiceName": "acm",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:acm:region:account-id:certificate/resource-id",
+  "Severity": "high",
+  "ResourceType": "AwsCertificateManagerCertificate",
+  "Description": "Check if ACM Certificates use a secure RSA key size",
+  "Risk": "Certificates with weak RSA keys can be compromised.",
+  "RelatedUrl": "https://docs.aws.amazon.com/es_es/acm/latest/userguide/acm-certificate.html",
```

Comment:
Please, can you add here the Security Hub link?
https://docs.aws.amazon.com/securityhub/latest/userguide/acm-controls.html#acm-2

Reply:
Added in commit 74786c73ecbc14a150b29733ea9fcda4c57d48ec